perceiver/data/audio/maestro_v3.py

```diff
@@ -0,0 +1,90 @@
+import functools
+import hashlib
+import json
+import os
+import shutil
+from pathlib import Path
+from typing import Any, Dict
+
+from perceiver.data.audio.symbolic import SymbolicAudioDataModule
+from perceiver.data.audio.utils import download_file, extract_file
+
+
+class MaestroV3DataModule(SymbolicAudioDataModule):
+    def __init__(
+        self,
+        *args: Any,
+        dataset_uri: str = "https://storage.googleapis.com/magentadata/datasets/maestro/v3.0.0/maestro-v3.0.0-midi.zip",
```

Comment:
A copy is now available at https://martin-krasser.com/perceiver/data/midi/maestro-v3.0.0-midi.zip